src/environments/environment.prod.ts

```diff
@@ -1,20 +1,34 @@
+import { HTTP_INTERCEPTORS } from '@angular/common/http';
 import { AuthConfig } from 'angular-oauth2-oidc';
+import { TransactionsInterceptor } from '../app/interceptors/transactions.interceptor';
+import { AccountsInterceptor } from '../app/interceptors/accounts-interceptor';
 
 export const environment = {
   production: true,
-  apiURL: '${API_URL}',
-  mockProviders: []
+  apiURL: '${API_ROOT}',
+  locales: '${LOCALES}'.split(','),
+  // TODO: Remove these interceptors when there is an api available
+  mockProviders: [{
+    provide: HTTP_INTERCEPTORS,
+    useClass: TransactionsInterceptor,
+    multi: true,
+  },
+  {
+    provide: HTTP_INTERCEPTORS,
+    useClass: AccountsInterceptor,
+    multi: true,
+  },]
 };
 
 export const authConfig: AuthConfig = {
   // Url of the Identity Provider
-  issuer: '${ISSUER}',
+  issuer: '${AUTH_URL}',
 
   // URL of the SPA to redirect the user to after login
-  redirectUri: window.location.origin + '/transactions',
+  redirectUri: '${PROTOCOL}//${HOSTNAME}:${PORT}${PATHNAME}${AUTH_LANDING_PAGE}',
```

Comment:
Do you think that this environment will be applied only for docker execution? Not for a local run?

Reply:
when we do `ng serve` it uses `environment.ts` but in the case of docker it will use production mode and take `environment.prod.ts` do you think it would be worth to have something like `environment.docker.ts`?

Reply:
I thought about that, but I think it's okay to leave it as is

Reply:
I'd revert this one ideed, as using window.location.origin is robust to using reverse proxies in between too...